static/src/js/components/GroupList/GroupList.jsx

```diff
@@ -191,7 +203,8 @@ const GroupList = () => {
       className: 'col-auto',
       dataAccessorFn: (members) => members.count
     },
-    {
+    // Hide the Actions column if the user does not have the correct permission
+    (!isAdmin || hasSystemGroup) && {
```

Comment:
Is it just the name of this that makes it confusing or is it wrong? 

If you're not an admin

OR

If you have system group access?